styles/account/account.ts

```diff
@@ -82,6 +82,13 @@ export default {
 
     },
 
+    newsletter: {
+        wrapper: 'col-start-1 col-span-2',
+        headline: 'mb-2 border-b border-gray-100 pb-2',
+        alert: 'mb-4 flex gap-4',
+        alertIcon: 'size-4 shrink-0',
```

Comment:
```suggestion
        outline: 'col-start-1 col-span-2',
        headline: 'mb-2 border-b border-gray-100 pb-2',
        alert: {
            outer: 'mb-4 flex gap-4',
            icon: 'size-4 shrink-0',
        },
```

i tried to use the same keys for similiar parts. maybe we can adjust it here?

Reply:
done